std/eval/_std/sys/ssl/Mbedtls.hx

```diff
@@ -0,0 +1,50 @@
+package sys.ssl;
+
+import mbedtls.Entropy;
+import mbedtls.CtrDrbg;
+
+class Mbedtls {
+	static public var entropy = new Entropy();
```

Comment:
Maaaybe this should be initialised lazily.

Reply:
Done in https://github.com/HaxeFoundation/haxe/commit/a9ec9d4c790a02e8e81879359f75252af769230f